tuist/tuist
# Add support for visionOS

### Short description 📝

Adds support for realityOS platform and devices

### How to test the changes locally 🧐

Here's how I did it:

```patch
diff --git a/Package.swift b/Package.swift
index 814cb0ce4..05536291b 100644
--- a/Package.swift
+++ b/Package.swift
@@ -62,7 +62,8 @@ let package = Package(
         .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", exact: "0.2.0"),
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", exact: "2.9.0"),
         .package(url: "https://github.com/SwiftGen/SwiftGen", exact: "6.5.1"),
-        .package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.10.0"),
+        // .package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.10.0"),
+        .package(name: "XcodeProj", path: "/Users/mstrodl/projects/xcodeproj"),
     ],
     targets: [
         .target(
diff --git a/Tuist/Dependencies.swift b/Tuist/Dependencies.swift
index a50361ded..c92825021 100644
--- a/Tuist/Dependencies.swift
+++ b/Tuist/Dependencies.swift
@@ -18,7 +18,8 @@ let dependencies = Dependencies(
         .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", .exact("0.2.0")),
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.9.0")),
         .package(url: "https://github.com/SwiftGen/SwiftGen", .exact("6.5.1")),
-        .package(url: "https://github.com/tuist/XcodeProj.git", .exact("8.10.0")),
+        // .package(url: "https://github.com/tuist/XcodeProj.git", .exact("8.10.0")),
+        .local(path: "/Users/mstrodl/projects/xcodeproj"),
     ],
     platforms: [.macOS]
 )
```
(Get a copy of my xcodeproj branch and update the path)

From there, you should be able to take a tuist project and replace the `platform: .iOS` with `platform: .visionOS`!

### Contributor checklist ✅

  - Updated the docstring, I think that's all I need?

### Reviewer checklist ✅
